### PR TITLE
LuaMocks: Add more items

### PR DESCRIPTION
**Pull request by badosu**

### CODE REVIEW

File: luamocks/SpringLuaApi.lua [hunk 1]
```diff
@@ -13,13 +13,60 @@
 --==================================================================================================
 --Variables for the MockUp
 Spring ={}
+Game = {}
+Engine = {}
+VFS = {}
+GL = {
+	SRC_ALPHA = stringMock,
+	ONE_MINUS_SRC_ALPHA = stringMock,
+	ONE = stringMock,
```
Comment (sprunk) [inline luamocks/SpringLuaApi.lua:22]: stringmock isn't yet defined

File: luamocks/SpringLuaApi.lua [hunk 2]
```diff
@@ -13,13 +13,60 @@
...
 stringMock ="TestString"
 tableMock ={exampletable= true}
+arrayMock = {}
 booleanMock =true
 functionMock =function (bar) return bar; end
 --==================================================================================================
 
+Game.armorTypes = arrayMock
+Game.gameID = numberMock
+Game.gameSpeed = numberMock
```
Comment (sprunk) [inline luamocks/SpringLuaApi.lua:37]: why not do like GL and just
```
Game = {
  armorTypes = arrayMock,
  gameID = numbermock,
 ....

Comment (badosu) [reply luamocks/SpringLuaApi.lua:37]: Can do!

**Comment by Beherith** [comment]
Scream when lgty :D

**Comment by badosu** [comment]
LGTM :+1:

**Pull request merged**